README.md

```diff
@@ -75,88 +81,138 @@ def fetchString(n: Int): Fetch[String] = Fetch(n) // or, more explicitly: Fetch(
 Now that we can convert `Int` values to `Fetch[String]`, let's try creating a fetch.
 
 ```scala
-import fetch.implicits._
 import fetch.syntax._
 
 val fetchOne: Fetch[String] = fetchString(1)
 ```
 
-Now that we have created a fetch, we can run it to a target monad. Note that the target monad (`Eval` in our example) needs to implement `MonadError[M, Throwable]`, we provide an instance for `Eval` in `fetch.implicits._`, that's why we imported it.
+Now that we have created a fetch, we can run it to a `Task`. Note that when we create a task we are not computing any value yet. Having a `Task` instance allows us to try to run it synchronously or asynchronously, choosing a scheduler.
```

Comment:
I don't see any reference to `Task` in these code snippets. (Not being familiar with the codebase) I'm assuming that `import fetch.implicits._` brings it in (or an implicit conversion from `Fetch` -> `Task`). It might be worth calling that out if so explicitly somewhere in the README.